lib/tasks/import.rake

```diff
@@ -0,0 +1,7 @@
+namespace :import do
+  desc 'Import twilio data for the either the specified DATE or yesterday'
+  task twilio: :environment do
+    date = ENV.fetch('DATE', Time.zone.yesterday.to_s)
```

Comment:
I think we should attempt to parse the user-provided date here too, otherwise we're needlessly scheduling a job when it's incorrect.